recipe/meta.yaml

```diff
@@ -53,10 +53,16 @@ requirements:
 test:
   imports:
     - xscen
+  source_files:
+    - tests
   commands:
     - pip check
+    - pytest tests
   requires:
     - pip
+    - pytest
+    - tox <4.0
+    - tox-conda >=0.10.2
```

Comment:
My bad, we don't bundle the tox configuration with the source code, so we can't leverage that here. I don't think we properly list our testing dependencies in the package itself. now that I'm looking (the only plugin I see is xdoctest).

We should add that to the package (in addition to a bunch of other changes; I'll open a PR).

Reply:
just revise the tox stuff for `xdoctest`. We'll cross our fingers, haha